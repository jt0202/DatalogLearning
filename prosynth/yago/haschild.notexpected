alfonso_xii_of_spain	olivia_winters
mary_de_bohun	prince_alexander_of_saxegessaphe
henri_chabot	breon_ocasey
isabella_i_of_castile	lt_Count_of_BarcelonaJuan_Carlos_I_of_Spaingt
alexandrine_of_mecklenburgschwerin	frederick_christian_elector_of_saxony
