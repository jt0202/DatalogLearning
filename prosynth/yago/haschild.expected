adam_chandler	gloria_marsh
babe_carey	jr_chandler
mindy_lewis	phillip_spaulding
jeffrey_oneill	olivia_spencer
nathan_hastings	olivia_winters
akhenaten	ankhesenamun
travis_montgomery	erica_kane
hayley_vaughan	mateo_santos
lizzie_spaulding	jonathan_randall
amenhotep_iii	sitamun
blair_cramer	todd_manning
harley_cooper	phillip_spaulding
jessica_buchanan	nash_brennan
joe_martin_all_my_children	ruth_martin_all_my_children
tad_martin	brooke_english
jennifer_rappaport	joey_buchanan
dixie_cooney_martin	adam_chandler
krystal_carey	david_hayward
ruth_martin_all_my_children	joe_martin_all_my_children
olivia_spencer	phillip_spaulding
natalia_rivera_aitoro	gus_aitoro
brooke_english	tad_martin
gus_aitoro	natalia_rivera_aitoro
akhenaten	meritaten
phillip_spaulding	olivia_spencer
dimitri_marick	erica_kane
phillip_spaulding	harley_cooper
todd_manning	t√©a_delgado
todd_manning	blair_cramer
jeff_martin_all_my_children	erica_kane
